docs/source/1.0/spec/aws/aws-cloudformation.rst

```diff
@@ -0,0 +1,672 @@
+=========================
+AWS CloudFormation traits
+=========================
+
+CloudFormation traits are used to describe Smithy resources and their
+components as they would be converted to `CloudFormation Resource Schemas`_.
```

Comment:
"so they can be" instead of "as they would be"?

Reply:
Can this also go into a little motivation? Like what you get by using these traits and leveraging your existing API definitions to build out resource schemas?